JavaScriptSDK.Interfaces/IConfiguration.ts

```diff
@@ -11,6 +11,37 @@ export interface IConfiguration {
     * Instrumentation key of resource
     */
     instrumentationKey: string; // todo: update later for multi-tenant?
+
+    /**
+     * Polling interval (in ms) for internal logging queue
+     */
+    diagnosticLoggingInterval?: number;
+
+    /**
+     * Maximum number of iKey transmitted logging telemetry per page view
+     */
+    maxMessageLimit?: number;
+
+    /**
+     * Console logging level
+     * 0: ALL console logging off
+     * 1: logs to console logs labelled at most as WARNING
+     * 2: logs to console logs laballed at most as CRITICAL

```

Comment:
would this be both warning and critical? How does user get both?

Reply:
```/**
     * Console logging level. All logs with a severity level higher
     * than the configured level will be printed to console. Otherwise 
     * they are suppressed. ie Level 2 will print both CRITICAL and
     * WARNING logs to console, level 1 prints only CRITICAL.
     * 
     * 0: ALL console logging off
     * 1: logs to console: severity >= CRITIAL
     * 2: logs to console: severity >= WARNING
     */
```
Updated IConfiguration in 1d827f0 to clarify loggingLevel configs